utils/utils.go

```diff
@@ -14,10 +14,10 @@ import (
 	"github.com/iotaledger/iota.go/v4/nodeclient/apimodels"
 )
 
-var UtilsLogger = evillogger.New("Utils")
+var log = evillogger.New("Utils")
 
 const (
-	MaxRetries    = 20
+	MaxRetries    = 100
```

Comment:
Would 100 times be too large? What about increasing `AwaitInterval` instead?

Reply:
It kept failing and failing, so I assumed it's better to wait longer than break Faucet requesting, but yeah let's try with 20 and 2 seconds